src/pages/what-is-ethereum.tsx

```diff
@@ -72,117 +61,103 @@ import stats from "@/public/images/upgrades/newrings.png"
 import hero from "@/public/images/what-is-ethereum.png"
 
 const Slogan = (props: ChildOnlyProp) => (
-  <Text
-    textStyle="normal"
-    fontWeight="normal"
-    fontSize="2rem"
-    lineHeight={1.4}
-    {...props}
-  />
+  <p className="text-[2rem] leading-xs" {...props} />
 )
 
 const Title = (props: ChildOnlyProp) => (
-  <Heading
-    as="h1"
-    fontSize="sm"
-    lineHeight={1.4}
-    letterSpacing="wider"
-    fontWeight="500"
-    mb={4}
-    textTransform="uppercase"
-    color="textTableOfContents"
+  <h1
+    className="text-sm font-medium uppercase leading-xs tracking-wider text-[#7f7f7f] dark:text-[hsla(0,0%,69.8%,.8)]"
```

Comment:
Validated with Nuno, lets use `body` color for this
```suggestion
    className="text-sm font-medium uppercase leading-xs tracking-wider"
```